core/src/main/java/org/apache/carbondata/core/datamap/dev/expr/DataMapExprWrapper.java

```diff
@@ -40,6 +41,18 @@
   List<ExtendedBlocklet> prune(List<Segment> segments, List<PartitionSpec> partitionsToPrune)
       throws IOException;
 
+  /**
+   * prune blocklet according distributable
+   *
+   * @param distributable     distributable
+   * @param partitionsToPrune partitions to prune
+   * @return the pruned ExtendedBlocklet list
+   * @throws IOException
+   */
+  List<ExtendedBlocklet> prune(DataMapDistributable distributable,
```

Comment:
the indent problem?

Reply:
there are other indent problems like this in this PR, better to optimize them all

Reply:
ok